packs/aws/sensors/sqs_sensor.py

```diff
@@ -0,0 +1,95 @@
+"""
+This is generic SQS Sensor using boto3 api to fetch messages from sqs queue.
+After receiving a messge it's content is passed as payload to a trigger 'aws.sqs_new_message'
+
+This sensor can be configured either by using config.yaml withing a pack or by creating
+following values in datastore:
+    - aws.input_queue
+    - aws.aws_access_key_id
```

Comment:
Ah, OK, I think I missed the datastore part.

It would also be good to explicitly document which values have precedence (config or datastore).